src/components/CourseDetails.js

```diff
@@ -18,40 +34,27 @@ CourseDetails.propTypes = {
     course: COURSE_MODEL,
 };
 
-const AwardTag = ({ award }) => {
-    if (award) {
-        return <Tag icon="university" mainText={award} />;
-    }
-
-    return null;
+const LevelAndAwardTag = ({ level, award }) => {
+    const tag = level || award ? <Tag topIcon="mortar-board" title={level} subText={award} /> : null;
+    return tag;
 };
```

Comment:
```suggestion
const LevelAndAwardTag = ({ level, award }) => {
    if (level || award) {
        return <Tag topIcon="mortar-board" title={level} subText={award} />
    }
    return null;
};
```
per Ferg's comment above.

Reply:
I think as we use the original style a bit more, it will become familiar and no longer difficult to parse. When learning Groovy, I found most of it's shortcuts difficult to read, but came to really like them as the language became familiar.

Reply:
It's not clear what the operator precedence is in the original line - does the `||` take precedence, or does the `?`?

Reply:
I prefer Richard's suggestion. It's not quite equivalent to the Groovy elvis operator because of the `||` - which is what justifies breaking it down.

Reply:
Annoyingly, if I put brackets around 'level || award' in the terniary version, Prettier removes them!